Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -144,6 +155,13 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
             collectionView.contentInset.bottom += typingIndicatorViewHeight
         }
         
+        view.addSubview(scrollToLatestMessageButton)
+        collectionView.bottomAnchor.pin(equalToSystemSpacingBelow: scrollToLatestMessageButton.bottomAnchor).isActive = true
+        scrollToLatestMessageButton.trailingAnchor.pin(equalTo: view.layoutMarginsGuide.trailingAnchor).isActive = true
+        scrollToLatestMessageButton.widthAnchor.pin(equalTo: scrollToLatestMessageButton.heightAnchor).isActive = true
+        scrollToLatestMessageButton.heightAnchor.pin(equalToConstant: 40).isActive = true
```

Comment:
nit: WDYT about having such constants in `intrinsicContentSize` ? 🤔 It can easily be customised by setting constraints on end-user side in VC / overriding the `intrinsicContentSize / overriding the `setupLayout` in component

```suggestion
        // in ScrollToLatestMessageButton
        override public var intrinsicContentSize: CGSize { .init(width: 40, height: 40) }
```

Reply:
@evsaev it's definitely an option and it requires a bigger discussion. The general approach is that the parent view decides on the size and position of its superview. The way it's done here is in sync with how we do it elsewhere.

Reply:
It's an interesting idea.
Pros: multiple customisation points
Cons: not consistent in our codebase, might be confusing for less experienced devs

Definitely something to consider. Not at this PR but more generally within the team